blog/fine-grained-news-2024-11.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Fine Grained News - November 2024
+description: Fine Grained News
+slug: fine-grained-news-2024-11
+date: 2024-11-30
+authors: hello-caleb
+tags: [newsletter]
+image: https://openfga.dev/img/og-rich-embed.png
+hide_table_of_contents: false
+---
+# Fine Grained News - November 2024
+
+Welcome to the November edition of Fine Grained News! As we enter the final stretch of 2024, there are exciting developments in the OpenFGA to share.
+
+🌟 We just hit 3,000 stars on the OpenFGA repo! A huge thank you to our incredible community for your support in building OpenFGA. Let's keep the momentum going! If you haven't yet, we'd greatly appreciate you [starring the repo](https://github.com/openfga/openfga) and pushing us toward 4,000 and growing our community! 🌟
+  <p align="center">
+  <img src="../static/img/blog/fgn-2024-11-stars.png" alt="OpenFGA 3000 stars" />
+</p>
+
+## Just Shipped
+
+**Batch Check API:**  We've launched Batch Check API in OpenFGA v1.8.0. Batching authorization checks together in a single request will significantly reduce roundtrip network latency. In addition, v.1.8.0 also added support for Contextual Tuples in the `Expand` API, and included other performance improvements. [Read more in the Changelog...](https://github.com/openfga/openfga/compare/v1.7.0...v1.8.0)
```

Comment:
```suggestion
**Batch Check API:**  We've launched Batch Check API in OpenFGA v1.8.0. Batching authorization checks together in a single request will significantly reduce roundtrip network latency. In addition, v.1.8.0 also added support for Contextual Tuples in the `Expand` API, and included other performance improvements. [Read more in the Changelog...](https://github.com/openfga/openfga/releases/tag/v1.8.0)
```

Link the actual changelog entry rather than the commits - it's more readable for folks, and they can reach the commits from there

Reply:
And let's keep this as 1.8.0, we're linking to 1.8.1 above and 1.8.1 doesn't have the batch check details

Reply:
I've listed v1.8.1 (because it's the latest release) and v1.8.0 (because of Batch Check). Let me know if we want to reduce the details or arrange it in some other way.